packages/bsync/src/index.ts

```diff
@@ -39,20 +46,25 @@ export class BsyncService {
   ): Promise<BsyncService> {
     const ac = new AbortController()
     const ctx = await AppContext.fromConfig(cfg, ac.signal, overrides)
-    const handler = connectNodeAdapter({
-      routes: routes(ctx),
-      shutdownSignal: ac.signal,
-    })
-    const server = http.createServer((req, res) => {
-      loggerMiddleware(req, res)
-      if (isHealth(req.url)) {
-        res.statusCode = 200
-        res.setHeader('content-type', 'application/json')
-        return res.end(JSON.stringify({ version: cfg.service.version }))
-      }
-      handler(req, res)
-    })
-    return new BsyncService({ ctx, server, ac })
+
+    const app = express()
+    app.use(cors({ maxAge: DAY / SECOND }))
```

Comment:
I don't think we need to support CORS on bsync since it's an implementation detail of the appview and end users don't end-up interacting with it.